app/Http/Controllers/Site/AirportController.php

```diff
@@ -26,24 +33,13 @@ public function show(Airport $airport)
             });
         })->collapse();
 
-        return $this->viewMake('site.airport.view')->with(['airport' => $airport, 'stations' => $stations, 'stands' => $this->loadStandStatus($airport)]);
-    }
-
-    private function loadStandStatus($airport)
-    {
-        $file_path = resource_path().'/assets/data/stands/'.strtolower($airport->icao).'.csv';
```

Comment:
Can you delete the old stand csv's in `resources/assets/data/stands` too

Reply:
Yes! d98d22a9